libs/api/libbpf_program.cpp

```diff
@@ -193,13 +190,25 @@ bpf_program__attach_xdp(struct bpf_program* program, int ifindex)
 }
 
 struct bpf_program*
-bpf_program__next(struct bpf_program* previous, const struct bpf_object* object)
+bpf_program__next(struct bpf_program* prev, const struct bpf_object* obj)
+{
+    return bpf_object__next_program(obj, prev);
+}
+
+struct bpf_program*
+bpf_object__next_program(const struct bpf_object* object, struct bpf_program* previous)
```

Comment:
I dont have any preference obj/object as long as its consistent in the source file.

Reply:
You're right, this should be obj/prev to match libbpf code as much as possible, so it can be removed as soon as we can use libbpf.c directly.  Updated.